docs/concepts/account-linking/index.mdx

```diff
@@ -3,80 +3,120 @@ title: Account Linking (FLIP 72)
 
-### Why Care?
+What makes account linking possible is an extension of the capabilities. Similar to how storage capabilities allow gaining access to a value stored in an account's storage, AuthAccount capabilities allow gaining access to an AuthAccount.
 
-Account Linking grants users access to their linked child accounts without needing to interface with the child account's
-custodial app, **and** the custodial app can interact with the relevant assets in the child account on behalf of the
-user in a frictionless UX free from transaction prompts.
+### Creating Account Links
 
-<Callout type="success">
+Creating an AuthAccount capability and publishing that capability to another account is what we define as account linking. An account owning the `AuthAccount` capability of another account is called a parent account, and the account sharing the capability with the other account becomes its child account.
+
+![Account linking on Flow](resources/linking.png)
+
+A link between two existing accounts on Flow can be created in two steps:
+
+1. a child account creates an `AuthAccount` capability and publishes it to the parent account.
+2. The parent account, claims that capability and can access the child's account through it.
+
+![Account linking steps on Flow](resources/linking-steps.png)
+
+These two steps are implemented in Cadence as two transactions:
+
+************************************Create capability************************************
+
+The account B creates and publishes the `AuthAccount` capability to the account A at the address `0x01`
+
+```json
+#allowAccountLinking
+
+transaction {
+    prepare(signer: AuthAccount) {
+        let capability = signer.linkAccount(/private/accountCapA)!
+        signer.inbox.publish(capability, name: "accountCapA", recipient: 0x1)
+    }
+}
+```
+
+****************************Claim capability****************************
+
+The account A claims the capability published by account B.
+
+```json
+transaction {
+    prepare(signer: AuthAccount) {
+        let capability = signer.inbox.claim<&AuthAccount>("accountCapB", provider: 0x2)!
+        let accountRef = capability.borrow()!
+    }
+}
+```
+
+## How is account linking useful?
+
+Account linking is very useful in the case of onboarding non-existing Flow users to your Flow application. Because participating on Flow network requires having a Flow account, which a non-web3-native user will struggle to achieve a lot of applications create the account for the user.
+
+The problem with this approach becomes when a user wants to use that account with another application, or move the items stored in that account out of the application. With account linking it’s possible for the application to create the account instead of the user, but still allow for the user to later link that account to any other accounts he will own.
```

Comment:
Want to avoid using terminology that implies a problem.

```suggestion
Naturally, users may expect to use their account with another application, or otherwise move assets stored in that account elsewhere. When an app initially leverages account linking it creates the account instead of the user and is where that user's specific state is stored. At a later point, users can take ownership of the app account providing they possess a full Flow account, typically by installing a wallet app.
```